pytest_postgresql/factories/process.py

```
@@ -94,6 +94,7 @@ def postgresql_proc(
     unixsocketdir: str | None = None,
     postgres_options: str | None = None,
     load: list[Callable | str | Path] | None = None,
+    load_autocommit: bool | None = None,
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)